src/gui/properties/propertieswidget.cpp

```diff
@@ -335,7 +336,16 @@ void PropertiesWidget::loadTorrentInfos(BitTorrent::Torrent *const torrent)
         m_ui->labelCommentVal->setText(Utils::Misc::parseHtmlLinks(m_torrent->comment().toHtmlEscaped()));
 
         m_ui->labelCreatedByVal->setText(m_torrent->creator());
+
+        // Private
+        m_ui->labelPrivateVal->setText(m_torrent->isPrivate() ? tr("Yes") : tr("No"));
     }
+    else
+    {
+        // Private (when there is no metadata)
```

Comment:
```suggestion
```

Reply:
It makes no sense to write obvious things in the comments, for example, **what** the code does (this is already clear from the code itself). If you want to explain it, then write down **why** it does it.

Reply:
P.S. Below is an example of a completely meaningless comment.
```c++
// Load dynamic data
loadDynamicData();
```

Reply:
Thanks... I agree with you that the comment is superfluous. I only added it because the code that is already there has the same type of comments.. 

Example: https://github.com/qbittorrent/qBittorrent/blob/d60c77ca6ab57e9f973db67ad4b55a14598b3827/src/gui/properties/propertieswidget.cpp#L321-L339